src/net.cpp

```diff
@@ -2733,6 +2768,7 @@ CNode::CNode(NodeId idIn, ServiceFlags nLocalServicesIn, int nMyStartingHeightIn
     addrName = addrNameIn == "" ? addr.ToStringIPPort() : addrNameIn;
     nVersion = 0;
     strSubVer = "";
+    m_prefer_evict = false;
```

Comment:
nit: Should use a C++11 default member initializer instead (That will also solve the merge conflict)